### ISSUE TITLE
Access keys don't work in the main menu

### ISSUE THREAD

**Issue by gabrbrand** [mjakeman/extension-manager#608]
**Describe the bug**
The recently added access keys for the main menu don't work. It seems to be a problem of libadwaita (or gtk 4), as it doesn't work in other libadwaita applications with access keys in the (main) menu either. It does however work in the GNOME Tweaks application (GNOME Shell 45.5, GTK 3.24.41).

**System Information:**
 - Distro: Fedora 39
 - Extension Manager version: 0.5.0
 - Package format: Flatpak

**Comment by oscfdezdz** [comment]
Yes, looks like a problem upstream. I've managed to see them by pressing <kbd>Tab</kbd> or arrow keys after <kbd>Alt</kbd> key.

**Comment by oscfdezdz** [comment]
Opened https://gitlab.gnome.org/GNOME/gtk/-/issues/6569, let's see how that goes.

**Comment by oscfdezdz** [comment]
The issue was resolved in GTK and should ship with GTK 4.18 and GNOME 48.
